public/index.html

```diff
@@ -15,6 +15,13 @@
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <link
```

Comment:
We already have a font awesome package installed. In order to avoid having too many packages/cdn can you use the one we have already?

Reply:
oh alright